src/dxgi/dxgi_factory.cpp

```diff
@@ -519,4 +549,15 @@ namespace dxvk {
   }
 
 
+  DXVK_VK_GLOBAL_HDR_STATE DxgiFactory::GlobalHDRState() {
+    std::unique_lock lock(s_globalHDRStateMutex);
+    return s_globalHDRState;
+  }
+
+
+  uint32_t DxgiFactory::GlobalHDRSerial() {
```

Comment:
Can we get rid of this function and just work with the struct?

Calling this separately works due to all the locking we have in place, but it a) forces us to lock twice and b) encourages patterns that can easily turn into a race condition if we ever change anything.

Reply:
Done.